reo/nested_inputs.py

```diff
@@ -102,6 +102,198 @@ def list_of_dict(input):
     result.append(dict(i))
   return result
 
+off_grid_defaults = {
+  "Scenario": {
+    "optimality_tolerance_techs": {
+      "type": "float",
+      "min": 1.0e-5,
+      "max": 10.0,
+      "default": 0.05,
+      "description": "The threshold for the difference between the solution's objective value and the best possible value at which the solver terminates"
+    },
+    "Site": {
+      "Financial": {
+        "om_cost_escalation_pct": {
```

Comment:
@adfarth some of the `off_grid_defaults` match the existing default values (such as for `om_cost_escalation_pct`). Can we remove the definitions for these keys from `off_grid_defaults`?

Reply:
Yes, I think we can. I left them just in case we wanted to make these defaults different from on-grid in the future, but we can always add them back in. Would you like me to remove all duplicates?

Reply:
Yes thank you

Reply:
I've removed all duplicates in the commit "Remove off-grid duplicates from nested_inputs.py"